### PR TITLE
Change all the references of `default_filter` to `human_friendly_id`

### PR DESCRIPTION
**Pull request by vsurresh**

### CODE REVIEW

**Comment by vsurresh** [comment]
Changed all the references from 'default_filter' to 'human_friendly_id: ["name__value"]'

PS - Assign it to petercrocker

File: models/examples/nautobot/nautobot-v2.yml [hunk 1]
```diff
@@ -275,7 +275,7 @@ nodes:
     description: "A Circuit represent a single physical link between two locations"
     label: "Circuit"
     icon: "mdi:transit-connection-variant"
-    default_filter: circuit_id__value
+    human_friendly_id: ["name__value"]
```
Comment (BaptisteGi) [inline models/examples/nautobot/nautobot-v2.yml]: Don't think circuit has a name, guess it should be "circuit_id__value"

File: models/examples/netbox/netbox.yml [hunk 2]
```diff
@@ -242,7 +242,7 @@ nodes:
     description: "A Circuit represent a single physical link between two locations"
     label: "Circuit"
     icon: "mdi:transit-connection-variant"
-    default_filter: circuit_id__value
+    human_friendly_id: ["name__value"]
```
Comment (BaptisteGi) [inline models/examples/netbox/netbox.yml]: Same as the above, it should be **circuit_id__value**

Comment (vsurresh) [reply models/examples/netbox/netbox.yml]: @BaptisteGi I've changed them all now.

File: models/examples/nautobot/nautobot-v2.yml (hunk 1 above)
Comment (vsurresh) [reply models/examples/nautobot/nautobot-v2.yml]: I've amended now

File: models/examples/netbox/netbox.yml (hunk 2 above)
Comment (petercrocker) [reply models/examples/netbox/netbox.yml]: Seems they're correct for every one but this one on line 245 of netbox.yml?

**Comment by wvandeun** [comment]
Still need to test infrahub-sync with updated netbox/nautobot schema

**Pull request closed without merging**